src/main/resources/application-local.yml

```diff
@@ -67,4 +67,7 @@ gridsuite:
     -
       name: voltage-init-server
       base-uri: http://localhost:5038
+    -
```

Comment:
Need to add in `application.yml` file

And `deployment/k8s/resources/study/config/study-server-application.yml`